mlflow/store/db_migrations/versions/27a6a02d2cf1_add_model_version_tags_table.py

```diff
@@ -0,0 +1,37 @@
+"""add model version tags table
+
+Revision ID: 27a6a02d2cf1
+Revises: 728d730b5ebd
+Create Date: 2020-06-26 13:30:27.611086
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+from mlflow.store.model_registry.dbmodels.models import SqlModelVersionTag
+
+revision = '27a6a02d2cf1'
+down_revision = '728d730b5ebd'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(SqlModelVersionTag.__tablename__,
```

Comment:
can we do both `CREATE TABLE` migrations for model version tag and registered models tag in the same migration script rather than creating two?

Reply:
I like the idea of using two separate migrations. If a unified migration fails half way, we're left with a phantom table that's unaccounted for by the migration script.